### DISCUSSION TITLE
How to disable physical keyboard shortcuts

### DISCUSSION THREAD

**Discussion by das-nibelung** [termux/termux-app#3951] [Q&A] [answered]
Hello, I'm using a physical keyboard. How do I disable the Alt + key combination that brings up the "more" menu for a second and then disappears?

**Comment by agnostic-apollo** [comment, marked as answer]
Add `disable-hardware-keyboard-shortcuts=true` to `~/.termux/termux.properties` file. It will disable all hardware shortcuts.

https://github.com/termux/termux-app/releases/tag/v0.118.0

**Reply by das-nibelung** [reply]
I changed properties file. This pop up menu still shows

**Reply by agnostic-apollo** [reply]
Did you follow the instructions in changelog afterwards?
